### ISSUE TITLE
nginx provides httpd

### ISSUE THREAD

**Issue by rlex** [oerdnj/deb.sury.org#641]
Hello, a little request. Official nginx packages have

```
Provides: httpd
```

Ubuntu packages also have 

```
Provides: httpd, httpd-cgi, nginx
```

In control. 

This is needed, in particular, to not install apache2 when running zabbix-frontend-web. Maybe some other cases when package expects webserver but not exact one.

Will be nice to have it in your packages as well.

**Comment by dromer** [comment]
yes, I already have nginx installed and do not want to have apache2 to go with my php :)
